broker-daemon/relayer/relayer-client.js

```diff
@@ -13,10 +13,19 @@ const consoleLogger = console
 consoleLogger.debug = console.log.bind(console)
 
 /**
+ * Path for Proto files for the Relayer when doing local development
  * @constant
  * @type {string}
  * @default
  */
+const LOCAL_RELAYER_PROTO_PATH = './proto-local/relayer.proto'
```

Comment:
@treygriffith what was the thought behind the local relayer path? Would we run into issues if we just updated the original `proto/relayer.proto` path?

Reply:
@dannypaz my goal was to still enable local development but without the risk of overwriting the file in version control. If we pulled into `proto/relayer.proto` it would update in version control and could be inadvertently committed.